### PR TITLE
[MIG] Merge product_brand_*_report into product_brand

### PR DESCRIPTION
**Pull request by chienandalu**
* [product_brand_sale_report](https://github.com/OCA/sale-reporting/tree/9.0/product_brand_sale_report) can now be merged into `product_brand` since this one meet the dependency of `sale`.
* The same goes for [product_brand_invoice_report](https://github.com/OCA/account-invoice-reporting/tree/9.0/product_brand_invoice_report)
* I moved the readme to the new OCA's guidelines as well.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: product_brand/README.rst
Comment (JayVora-SerpentCS) [inline product_brand/README.rst:67]: Keep text only upto feedback.

File: product_brand/__manifest__.py
Comment (JayVora-SerpentCS) [inline product_brand/__manifest__.py:15]: Make it Serpent Consulting Services Pvt. Ltd. (so its added to right author on search)

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: product_brand/README.rst
Comment (pedrobaeza) [reply product_brand/README.rst:67]: @JayVora-SerpentCS this text is now auto-generated by the script, so it's correct and self-mantained.

Comment (JayVora-SerpentCS) [reply product_brand/README.rst:67]: I see, I was referring to usual OCA readme format though.

Comment (pedrobaeza) [reply product_brand/README.rst:67]: This is the new OCA README format :wink:

You can see the new README in the template that states the automatic generation: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Review by rafaelbn — APPROVED** [review]
Tested functionally :+1: 

Please @JayVora-SerpentCS could unblock the merge? :smile:

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
Approved!

**Comment by pedrobaeza** [comment]
You need to make the migration part that merges module on OpenUpgrade, @chienandalu

**Pull request merged**
